types/bootstrap-growl-ifightcrime/index.d.ts

```diff
@@ -47,9 +47,9 @@ declare namespace BootstrapGrowlIfightcrime {
          align?: AlignType;
 
         /**
-         * Integer
+         * Integer or 'auto'
          */
-         width?: number;
+         width: number | 'auto';
```

Comment:
The width attribute still optional, because has been declared in default options.